types/types.go

```diff
@@ -1033,6 +1033,45 @@ func (sig Signature) MarshalText() ([]byte, error) { return marshalHex("sig", si
 // UnmarshalText implements encoding.TextUnmarshaler.
 func (sig *Signature) UnmarshalText(b []byte) error { return unmarshalHex(sig[:], "sig", b) }
 
+// MarshalJSON implements json.Marshaler.
+func (fcr FileContractRevision) MarshalJSON() ([]byte, error) {
```

Comment:
This feels like it’d be better to separate the types over the doc string that mentions the sentinel value and custom Marshal methods to ignore the field

Reply:
I don't follow...?

Reply:
Apparently I’m just crazy’

Reply:
😆 

It *is* slightly gross to embed a type and special-case one of its fields, but arguably this is just a mistake in Sia v1 -- revisions *should* embed the entire contract. A consensus rule can enforce that the payout value doesn't change. v2 fixes this :tm: